shared/cfg/base.cfg

```diff
@@ -268,6 +268,17 @@ image_unbootable_pattern = "Hard Disk.*not a bootable disk"
 # export_ip: optional.
 # export_options: optional.
 
+# SAN support related params. Please fill the params based on the environment
```

Comment:
@balamuruhans could you check if we can reuse this param `image_raw_device`, in-additional , so that even qemu   vm can get benefited...and basically `image_raw_device` can be replaced with `storage_lun_id` and we can give the full path instead, may be sometime we might have to `/dev/mapper` so lets not hardcode `/dev/disk/by-id`

Reply:
sure, I will check to reuse the image_raw_device and avoid hardcoding /dev/disk/by-id. Thank you.

Reply:
Thanks for the improvements, made the changes as suggested. I will test it and post the results for the same